scanner/image/scanner/scripts/save-dir-contents

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ $# -lt 1 ]; then
+	echo >&2 "Usage: $0 <dirs...>"
```

Comment:
Very minor, maybe for someone who's not familiar with the script, we can add:
```
echo >&2 "Error: At least one directory must be provided."
echo >&2 "Usage: $0 <dirs...>"
```

Reply:
This file matches https://github.com/stackrox/stackrox/blob/master/image/static-bin/save-dir-contents and I don't want to modify it unless we absolutely have to